conf/functions_worker.yml

```diff
@@ -437,3 +437,4 @@ zooKeeperSessionTimeoutMillis: -1
 # ZooKeeper operation timeout in seconds
 # Deprecated: use metadataStoreOperationTimeoutSeconds
 zooKeeperOperationTimeoutSeconds: -1
+connectorsCatalogueUrl: file:///Users/nicolo.boschi/catalogue.yaml
```

Comment:
this catalogue will include URLs that will probably refer to third party websites.

It may be better to also configure a list of allowed DNS names for the final connector URLs,
this way a forged catalogue cannot force the worker to download anything from the Internet.

In other words we must have an explicit list of allowed sources for the .nar files. It can be a comma separated list of DNS names or "base urls"

connectorsAllowedDownloadArchives: https://server1/xxx,https://server2/xxx,https://server2/yyyyy

(the comma character is not common for URLs, we can use it without problems